.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: java
```

Comment:
This should allow it to be built in Travis which gets us another CI environment to spot issues etc.